packages/drawer/src/views/Drawer.tsx

```diff
@@ -92,6 +92,7 @@ type Props = {
   renderDrawerContent: Renderer;
   renderSceneContent: Renderer;
   gestureHandlerProps?: React.ComponentProps<typeof PanGestureHandler>;
+  isBigScreen?: boolean;
```

Comment:
We don't need a `isBigScreen` prop. We can pass something like `drawerType={screenWidth > 800 ? 'sidebar' : 'front'}`